### DISCUSSION TITLE
Error: `rescue in convert': Can't find stylesheet to import. (Jekyll::Converters::Scss::SyntaxError) raise SyntaxError, e.message

### DISCUSSION THREAD

**Discussion by Gaur4vGaur** [cotes2020/jekyll-theme-chirpy#2137] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Built from `jekyll-theme-chirpy`

### Description

I merged 7.2.3 release to my fork and got the below error

Error: Can't find stylesheet to import.
  ╷
1 │ @use 'vendors/bootstrap';
  │ ^^^^^^^^^^^^^^^^^^^^^^^^
  ╵
  main.bundle.scss 1:1                                                                                 @use
  /home/runner/work/Gaur4vGaur.github.io/Gaur4vGaur.github.io/assets/css/jekyll-theme-chirpy.scss 1:1  root stylesheet 
  Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/jekyll-theme-chirpy.scss':
                    Can't find stylesheet to import.
                    ------------------------------------------------
      Jekyll 4.3.4   Please append `--trace` to the `build` command 
                     for any additional information or backtrace. 
                    ------------------------------------------------
/home/runner/work/Gaur4vGaur.github.io/Gaur4vGaur.github.io/vendor/bundle/ruby/3.3.0/gems/jekyll-sass-converter-3.0.0/lib/jekyll/converters/scss.rb:175:in `rescue in convert': Can't find stylesheet to import. (Jekyll::Converters::Scss::SyntaxError)
        raise SyntaxError, e.message
              ^^^^^^^^^^^^^^^^^^^^^^
	from /home/runner/work/Gaur4vGaur.github.io/Gaur4vGaur.github.io/vendor/bundle/ruby/3.3.0/gems/jekyll-sass-converter-3.0.0/lib/jekyll/converters/scss.rb:[15](https://github.com/Gaur4vGaur/Gaur4vGaur.github.io/actions/runs/12411749849/job/34650089787#step:5:16)9:in `convert'

### Operations you have already tried

It seems like a syntax error in below files

https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/assets/css/jekyll-theme-chirpy.scss

---
---

@use 'main
{%- if jekyll.environment == 'production' -%}
  .bundle
{%- endif -%}
';

/* append your custom style below */

### Anything else?

failed job - https://github.com/Gaur4vGaur/Gaur4vGaur.github.io/actions/runs/12411749849/job/34650089787

**Comment by SoumyaK4** [comment, marked as answer]
Make sure you're following everything in [this doc](https://github.com/cotes2020/jekyll-theme-chirpy/wiki/Upgrade-Guide#upgrade-the-fork)

also another suggestion, if you're not contributing to chirpy, it's better to use the starter as there's less complications for upgrading
